client/project.go

```diff
@@ -360,8 +360,24 @@ func (c *Client) VerifyDomain(projectID, domainName string) error {
 	return c.request("POST", path, nil, nil, nil)
 }
 
-// ProvisionCertificate creates a valid TLS certificate for a custom domain name.
-func (c *Client) ProvisionCertificate(projectID, domainName string) error {
+// ProvisionCertificateAutomatic automatically provisions a valid TLS
+// certificate for a custom domain name.
+func (c *Client) ProvisionCertificateAutomatic(projectID, domainName string) error {
 	path := fmt.Sprintf("/api/projects/%s/domains/%s/certificates", projectID, domainName)
-	return c.request("POST", path, nil, nil, nil)
+	bs, err := json.Marshal(map[string]string{"strategy": "automatic"})
```

Comment:
Can we specify the body as a struct instead?

Reply:
On second thought, let's not -- this is a one-off struct only used to send the request, it's never used beyond this point, and since the automatic and manual strategies have different bodies, it would be an awkward struct to make without adding validation steps to the request or splitting into multiple structs and using composed structs/promoted fields and I don't wanna go down that route.